build/yamls/base/conf/antrea-agent.conf

```diff
@@ -241,3 +241,9 @@ antreaProxy:
   # Note that setting ProxyLoadBalancerIPs to false usually only makes sense when ProxyAll is set to true and
   # kube-proxy is removed from the cluser, otherwise kube-proxy will still load-balance this traffic.
   #proxyLoadBalancerIPs: true
+
+# The role in which Antrea Agent is working. Supported values:
+# - Node (default)
+# - VM
```

Comment:
Any difference between VM and BM and so we need two types? If not or not clear, let us just add one type called "VirtualMachine" for now.